lang/lang.txt

```diff
@@ -1854,7 +1854,7 @@ pt=Latência do CAS da memória principal atual.
 it=Latenza CAS della memoria principale corrente.
 nl=Huidig hoofdgeheugen CAS Latency.
 da=Aktuel hovedhukommelse CAS Latency.
-zh-CN=当前主存 CAS 延迟。
+zh-CN=当前主内存的 CAS 延迟。; CL
```

Comment:
Is the text after ; intended as comment? If yes, please remove it.

Reply:
Yes, they are comments that I forgot to remove before. Now they have been removed.